mmdet3d/__init__.py

```diff
@@ -19,7 +19,7 @@ def digit_version(version_str):
 
 
 mmcv_minimum_version = '1.5.2'
-mmcv_maximum_version = '1.7.0'
+mmcv_maximum_version = '1.8.0'
```

Comment:
Please modify the version of MMCV at https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/faq.md and https://github.com/open-mmlab/mmdetection3d/blob/master/docs/zh_cn/faq.md

Reply:
done.

Reply:
why not this change sync to master branch? I have a little doubt.

Reply:
we usually merge pr into dev first, then merge dev into master.

Reply:
The dev branch has been modified to mmcv_maximum_version=1.8.0. So this PR is useless. Just need to synchronize dev to master later. thanks~